### PR TITLE
[255215] Detect URL with empty path

### PR DESCRIPTION
**Pull request by Double-oxygeN**
https://acsmine.tok.access-company.com/redmine/issues/255215

### CODE REVIEW

**Review by irisTa56 — COMMENTED** [review]
Is this a temporary change for debugging purpose?
If so, leaving a TODO comment might be good so that we don't forget to delete it.

File: lib/util/httpc.ex [hunk 1]
```diff
@@ -135,6 +137,10 @@ defmodule Antikythera.Httpc do
 
           %URI{uri | query: query_string} |> URI.to_string()
       end
+      |> do_if_url_has_empty_path(fn parsed_uri ->
```
[the remark this answers is not in the document]
Comment (Double-oxygeN) [reply lib/util/httpc.ex]: Thank you for your suggestion! I did not know about the feature of `Kernel.SpecialForms.with/1` when it fails clause-matching. It fits the situation better.

Fixed at 10db3ea0ab6d9073e99707d2330d2ed864b16bca.

**Comment by Double-oxygeN** [comment]
TODO comment might not be necessary for it.
This feature is truly for debugging, but not sure it is temporary.
It is better not to delete it until the problem is likely to be resolved.

**Comment by irisTa56** [comment]
> TODO comment might not be necessary for it. This feature is truly for debugging, but not sure it is temporary. It is better not to delete it until the problem is likely to be resolved.

Thanks. I got it.

**Review by irisTa56 — APPROVED** [review]
LGTM!

**Review by aMasakiTakahashi — APPROVED** [review]
LGTM!

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

**Pull request merged**